packages/source-storybook/src/index.ts

```diff
@@ -0,0 +1,122 @@
+import { Observable, forkJoin, timer } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import HttpsProxyAgent from 'https-proxy-agent';
+import { z } from 'zod';
+import type { Source } from '@jpmorganchase/mosaic-types';
+import { sourceScheduleSchema, validateMosaicSchema } from '@jpmorganchase/mosaic-schemas';
+import { fromHttpRequest, isErrorResponse } from '@jpmorganchase/mosaic-from-http-request';
+import { StoriesResponseJSON, StorybookPage, StoryConfig } from './types/index.js';
+
+// @ts-ignore
+export const schema = z.object({
+  schedule: sourceScheduleSchema.optional(),
+  prefixDir: z.string(),
+  proxyEndpoint: z.string().url().optional(),
+  stories: z.array(
+    z.object({
+      description: z.string(),
+      url: z.string(),
+      additionalData: z.object({}).passthrough().optional(),
+      additionalTags: z.array(z.string()).optional(),
+      filter: z
+        .any({ required_error: 'A regex is required to filter stories' })
```

Comment:
required error on an optional?

Reply:
ah yes... it was required and then I added filterTags as well... good shout!
Will remove